powerauth-push-server/src/main/java/io/getlime/push/service/AppRelatedPushClientCacheLoader.java

```diff
@@ -65,22 +65,23 @@ public AppRelatedPushClient reload(final String appId, final AppRelatedPushClien
         }
 
         logger.debug("LastUpdated differs for app: {}", appId);
-        return load(appId);
+        return createPushClient(credentials);
     }
 
     @Override
-    public AppRelatedPushClient load(final String key) throws Exception {
-        final AppCredentialsEntity credentials = appCredentialsRepository.findFirstByAppId(key).orElse(null);
+    public AppRelatedPushClient load(final String appId) throws Exception {
+        final AppCredentialsEntity credentials = appCredentialsRepository.findFirstByAppId(appId).orElse(null);
         if (credentials == null) {
-            logger.debug("AppCredentials does not exist for app: {}", key);
+            logger.debug("AppCredentials does not exist for app: {}", appId);
```

Comment:
I suggest raising it as a warning, as this might not be a breaking problem, but very probably it can be. Same for reload.